harness/ts/tsconfig.json.tmpl

```diff
@@ -0,0 +1,20 @@
+{
+  "extends": "@tsconfig/node16/tsconfig.json",
+  "version": "4.4.2",
+  "compilerOptions": {
+    "baseUrl": ".",
+    "outDir": "./tslib",
+    "rootDirs": ["../", "."],
+    "paths": {
+      "@temporalio/harness": ["../harness/ts/harness.ts"],
+
+      "*": ["node_modules/*", "node_modules/@types/*"]
+    },
+    "typeRoots": ["node_modules/@types"],
```

Comment:
Why do we need this?

Reply:
When using local linking NPM doesn't actually install transitive dependencies properly for some crazy reason, and then on top of that because of way I have to point to a different dir here it just doesn't search for `node_modules` in the right place, and even when you specify just that, it still fails to find types properly. :man_shrugging:  node.